docs/section5/Rule5-37.md

```diff
@@ -0,0 +1,64 @@
+
+# Envelope - Rule 5-37  
+
```

Comment:
Confuding mismatch in rule # here vs in the title above (Add Rule5-31)

Reply:
PR number is now updated to Rule 5-37.